frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,296 @@
+      span.c-rect
+
+  i18n.has-text-1.c-invite-description(tag='p') Here's a list of all invite links you own
+
+  table.table.c-table(v-if='activeList.length')
+    thead
+      tr
+        i18n.c-name(tag='th') created for
+        i18n.c-invite-link(tag='th') invite link
+        i18n.c-state(tag='th') state
+        th.c-action(
+          :aria-label='L("action")'
+        )
+    tbody
+      tr(
+        v-for='(item, index) in activeList'
+        :key='index'
+      )
+        td.c-name
+          | {{ item.name }}
+          tooltip.c-name-tooltip(
+            direction='top'
+            :isTextCenter='true'
+            :text='L("This invite link was only available during the onboarding period.")'
+          )
+            button.is-icon-small(v-if='item.isAnyoneLink')
+              i.icon-info-circle
+        td.c-invite-link
+          copy-to-clipboard.c-invite-link-wrapper(:textToCopy='item.inviteLink')
+            span.link.has-ellipsis {{ item.inviteLink }}
+            button.is-icon-small.has-background.c-invite-link-button
+              i.icon-copy.is-regular
+        td.c-state
+          i18n.c-state-description {{ item.state.description }}
+          i18n.c-state-expire(
+            v-if='item.state.expireInfo'
+            :class='item.state.isExpired ? "expired" : ""'
+          ) {{ item.state.expireInfo }}
+        td.c-action
+          invite-action-button.c-invite-action-button(:linkExpired='item.state.isExpired')
+  .c-empty-list(v-else)
+    SvgInvitation
+
+  // TODO: router-link doesn't work and it might be related to the fact that this p tag is the output result of i18n.vue component.
+  // Discussion needed to solve the problem.
+  i18n.c-invite-footer(
+    tag='p'
+    html='To generate a new link, you need to <router-link class="link" to="/invite">propose adding a new member</router-link> to your group.'
+  )
+</template>
+
+<script>
+import PageSection from '@components/PageSection.vue'
+import Tooltip from '@components/Tooltip.vue'
+import SvgInvitation from '@svgs/invitation.svg'
+import InviteActionButton from './InviteActionButton.vue'
+import CopyToClipboard from '@components/CopyToClipboard.vue'
+
+export default {
+  name: 'InviteLinks',
+  components: {
+    PageSection,
+    SvgInvitation,
+    InviteActionButton,
+    Tooltip,
+    CopyToClipboard
+  },
+  data () {
+    return {
+      ephemeral: {
+        selectbox: {
+          focused: false,
+          selectedOption: 'Active'
+        }
+      }
+    }
+  },
+  methods: {
+    unfocusSelect () {
+      this.$refs.select.blur()
+    }
+  },
+  computed: {
+    activeList () {
+      return this.ephemeral.selectbox.selectedOption === 'Active' ? this.list.filter(item => !item.state.isExpired)
+        : this.ephemeral.selectbox.selectedOption === 'All' && this.list
+    }
+  },
+  props: {
+    list: {
+      type: Array
+      // TODO: leaving a comment explaining the structure of this prop.
```

Comment:
Can you remove this property as well? Most of the time we have components fetch directly from the vuex store instead of passing props around